machine-learning/05-machine_learning_exercise.Rmd

```diff
@@ -0,0 +1,245 @@
+---
+title: "Exercise: Exploring drug exposure in the NCI-60 cell lines"
+output: 
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+## NCI-60 Cell Line Data
+
+In this exercise, we'll work with datafrom GEO accession [`GSE116436`](https://www.refine.bio/experiments/GSE116436/drug-induced-change-in-gene-expression-across-nci-60-cell-lines-after-exposure-to-15-anticancer-agents-for-2-6-and-24h)
+that was processed with refine.bio. 
+
+Here's an excerpt of the description from the authors:
+
+> To identify patterns of drug-induced gene modulation that occur across different cell types, we measured gene expression changes across NCI-60 cell lines after exposure to 15 anticancer agents.
+
+**We've supplied the following to you:**
+
+* The gene expression matrix (`data/GSE116436/GSE116436_filtered.tsv`)
+* The sample metadata (`data/GSE116436/metadata_GSE116436_cleaned.tsv`)
+* Sample-sample Pearson correlation matrix (`data/GSE116436/GSE116436_sample_correlation.RDS`)
+* PCA (`prcomp` output) (`models/NCI60_prcomp_results.RDS`)
+* A PLIER model (`models/NCI60_PLIER_model.RDS`)
+
+_Note that we've only made a subset of samples available and we've already
```

Comment:
```suggestion
_Note that we've only processed a subset of the samples and we've already
```

Reply:
This sentence is going to get taken out.